api/v1/helpers/verbs/postUtils.js

```diff
@@ -65,14 +60,6 @@ function handlePostResult(o, resultObj, props, res, req) {
       realtimeEvents.sample.add, props.associatedModels.aspect);
```

Comment:
@shriramshankar do we need the publish code here? Will samples code ever reach api/v1/helpers/verbs/postUtils.js?

Reply:
Nice catch. This is not required.